terraform-build-modules/intro.md

```diff
@@ -0,0 +1,13 @@
+# Build a Module
+
+This Katacoda scenario is meant to serve as a companion piece for the 
+[Build a Module](https://learn.hashicorp.com/tutorials/terraform/module-use?in=terraform/modules)
+Terraform Learn tutorial.
+
+In this scenario, you will create a module to manage AWS S3 buckets used to host static websites.
+
+> This environment has been pre-configured with the Terraform CLI, AWS CLI and 
+[localstack](https://localstack.cloud/), a mocking framework that provides a 
+testing environment on your local machine that provides the same functionality 
+and APIs as the real AWS cloud environment. As a result, you do not need an AWS
+account to complete this scenario.
```

Comment:
```suggestion
> This environment has been pre-configured with the Terraform CLI, AWS CLI and 
[localstack](https://localstack.cloud/), a testing framework that mocks core AWS APIs locally. As a result, you do not need an AWS
account to complete this scenario.
```

Reply:
I updated it to the following because localstack is **not** a testing framework :x

> This environment has been pre-configured with the Terraform CLI, AWS CLI and [localstack](https://localstack.cloud/), a mocking framework that mocks core AWS APIs locally. As a result, you do not need an AWS account to complete this scenario.